Bugfix: dumps the object which has the json fields in bulk create

ponytailer opened the pull request:
- bugfix for  `orjson.dumps -> bytes,  but expected str`
- define the `__queryset_cls__`

BTW,  `merge and squash` may be better.

collerek commented:
What is the bug that this PR fixes?

ponytailer commented:
> What is the bug that this PR fixes?

when use the bulk_create and object has the json field, 

`save_mixins.dump_all_json_fields_to_str: orjson.dumps return the bytes, not the str`

collerek commented on tests/test_queries/test_utils.py:
```diff
@@ -0,0 +1,15 @@
+import json
```
This doesn't test orjson

collerek commented on ormar/queryset/utils.py:
```diff
@@ -17,6 +17,15 @@
     from ormar import Model, BaseField
 
 
+def to_str(val: Union[bytes, str, int]) -> str:
```
There already is:

https://github.com/collerek/ormar/blob/8dc05d5cf48469f6285e88ab87d38b57c678976f/ormar/fields/parsers.py#L40-L43

which probably can be re-used

ponytailer replied:
All use the `to_str`.

collerek commented on tests/test_queries/test_queryset_level_methods.py:
```diff
@@ -76,6 +78,27 @@ class Meta:
+                raise ValueError("customer not found")
+            return entity
+
+    __queryset_cls__ = QuerySetCls
```
This should be added to docs in sections of models as a way to overwrite setting for main/default `QuerySet`

collerek commented:
Thanks for catching the bug and this PR!

I refactored to avoid code duplication.

Also, all model configs should go into `model.Meta` so I moved the `queryset_class` config parameter.

ponytailer commented:
> Thanks for catching the bug and this PR!
> 
> I refactored to avoid code duplication.
> 
> Also, all model configs should go into `model.Meta` so I moved the `queryset_class` config parameter.

ok, got it.

Release the version for the bugfix, please.
